Makefile

```diff
@@ -334,20 +360,12 @@ deploy/integreatly-rhmi-cr.yml:
 .PHONY: prepare-patch-release
 prepare-patch-release:
 	$(CONTAINER_ENGINE) pull quay.io/integreatly/delorean-cli:master
-	$(CONTAINER_ENGINE) run --rm -e KUBECONFIG=/kube.config -v "${HOME}/.kube/config":/kube.config:z -v "${HOME}/.delorean.yaml:/.delorean.yaml" quay.io/integreatly/delorean-cli:master delorean release openshift-ci-release --config /.delorean.yaml --version $(TAG)
+	$(CONTAINER_ENGINE) run --rm -e KUBECONFIG=/kube.config -v "${HOME}/.kube/config":/kube.config:z -v "${HOME}/.delorean.yaml:/.delorean.yaml" quay.io/integreatly/delorean-cli:master delorean release openshift-ci-release --config /.delorean.yaml --olmType $(olmType) --version $(TAG)
 
 .PHONY: release/prepare
 release/prepare:
 	@./scripts/prepare-release.sh
 
-.PHONY: push/csv
```

Comment:
I've been using this recently. I think there was some discussion that we don't use it anymore. Is there an alternative way? If not can we leave this in?

Reply:
With the time constrain I have added this back in. Not sure if it was updated correctly to work with but rhmi and rhoam. It's some thing I don't use so not sure the best way to test.

Reply:
By the way, it's possible I only used this because of some legacy knowledge I have about installing RHMI. If there is another way that this is to be used, I'd like to learn about it. 😄